package.json

```diff
@@ -48,6 +48,7 @@
     "test": "lerna run --stream test",
     "test.watch": "lerna run --stream test.watch",
     "ci-build-no-rust": "lerna run --ignore @temporalio/core-bridge --stream build",
+    "ci-per-change-load": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url='' --max-cached-wfs 500",
     "ci-load": "node ./packages/test/lib/load/run-all-stress-ci-scenarios.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url='' --max-cached-wfs 500",
```

Comment:
Why not run all of the stress scenarios that we previously ran (`ci-load`)?
I'd consider running them in parallel to save some time.

Reply:
That just changed recently though, no? It looks like they take quite a while to run. We could parallelize them and keep it as part of the PR pipeline if there are a small enough number.

It also looks like the self-hosted worker isn't really autoscaling. So we'd need to figure that out before changing things to work that way. I'm thinking we can follow up with another PR